docs/user_guide.md

```diff
@@ -39,108 +41,144 @@ Sample usage:
 - The timelines (blue bar) of the charts should be aligned, so that the comparison of contribution between two authors are easier.
 - There is no limit to the area of the ramp. If the contribution for a time period is too large compared to the rest of the time period, it is going to overlap with the neighbor ramps. Thus, the ramps are transparent, so they will not cover their neighbors.
-- As Figure shown, when the user hovers the mouse above a ramp, the time period and the exact amount of contribution will be shown.
-- If you click on a ramp, a github page containing the commits in that period of time will be opened
+- As Figure shown, when the user hovers the mouse a ramp, the time period and the exact amount of contribution will be shown.
+- If you click on a ramp, a GitHub page containing the commits in that period of time will be opened.
+
+#### Total Contribution Bars
+The total amount of code contributed is represented by the **red bars**, and the length of these red bars is proportional to the total contribution of the corresponding author.
+Hovering over the bar shows the exact amount of contribution.
+If the author contributes **too much** compared to other authors in the report, there will be multiple red bars for him.
+
 
 ### Code Panel
-The Code Panel allows the instructors to review students' code in the report, and easily maps a line to its author.
-If the user clicks on the name of the author in the visualizations of Chart Panel, the Code Panel will slide from right.
+The `Code Panel` allows users to review contributers' code in the report, showing all the lines written by the selected author.
+Clicking on the name of the author, in the `Chart Panel`, will display the `Code Panel` on the right.
 
 Below is the list of features in this panel:
-- All files that contain author's contribution will be shown in this panel. The file that contains more lines written by the author will be shown on the top.
-- After the user has reviewed a file, he can click the title of the file to hide it.
-- The user can click the title again to display the file content.
-- The lines that are NOT written by the current author will be marked gray, but they will still be displayed to provide context for the user
-- Segments of codes that are not written by the current author is default to be collapsed. If the user wants to refer to it, he can open the segment.
+- Files that contain author's contribution will be shown in this panel, sorted by the number of lines written.
+- Clicking the file title will show/hide the file content.
+- The lines that are NOT written by the selected author will be marked in gray and displayed to provide context for the user.
+- Segments of codes that are not written by the selected author are by default collapsed. User can click on the segments to display them, if necessary.
```

Comment:
This 2 lines are kind of weird. 
We 1st mention that the lines not written are displayed (but marked in grey), but the 2nd line mention that the segments of codes not written are collapsed (contradicts the previous line)? 
Maybe we can mix this 2 lines together, and mention that the codes are collapsed 1st then mention they will be marked in grey if displayed?